### ISSUE TITLE
Hey, just a minor typo

### ISSUE THREAD

**Issue by Milton3450** [DennisWG/Roid-Macros#30]
Hey, Super Nice addon good work. I just found a typo here https://denniswg.github.io/Roid-Macros/commands/equip/ .

The macro is: /equipoh, but in reality that command doesn't work, but i found /equipoff to work instead by guessing. So just a minor fix to the documentation :)

**Comment by DennisWG** [comment]
It is `/equipoh`, though.

See: [Console.lua line 24](https://github.com/DennisWG/Roid-Macros/blob/8af76f5f1bd463138bccf7fac16e0fcc144d81f3/Console.lua#L24)

There isn't a single occurrence of `equipoff` in the entire project and there never was. See: [3d95b7e](https://github.com/DennisWG/Roid-Macros/commit/3d95b7eea500ee2f0dfb97bcba1b985e579a1c02#diff-6d06b2f944ee7e05392630cf350399eeR806).

Are you sure you're not using another addon that provides this functionality?

**Comment by DennisWG** [comment]
Is it still not working if you only login with Roid-Macros enabled? If it is, try turning your addons back on again one at a time and see which one causes this issue and please report back to me.

**Comment by Milton3450** [comment]
You are right. I tried disabling all addons and now only /equipoh works. So it must have been an addon that interefered, dunno which one.

**Comment by DennisWG** [comment]
Try enabling them again one at a time and see after which one it stops working. If you can tell me the name of the addon I might be able to make my addon compatible with it.

**Comment by Milton3450** [comment]
Very odd: I turned all addons back and /equipoh is still the one working. I don't know what caused it.
